src/gwproto/data_classes/cacs/rest_poller_cac.py

```diff
@@ -0,0 +1,5 @@
+from gwproto.data_classes.component_attribute_class import ComponentAttributeClass
+
+
+class RESTPollerCac(ComponentAttributeClass):
```

Comment:
Not sure why this exists? What would be a rest poller other than our SCADA box?

Reply:
REST polling can be used with anything that implements a converter function between the reply returned and the message forwarded from that to scada. All the other parameters for REST polling can be implemented via the configuration in RESTPollerSettings, and those can be placed in a component, which needs a cac.